### ISSUE TITLE
Something really funny

### ISSUE THREAD

**Issue by eeyrw** [huggingface/pytorch-pretrained-BigGAN#5]
When I tried to generate image of a cat...
![output_1](https://user-images.githubusercontent.com/2088817/54967872-b8391600-4fb3-11e9-8e41-87b17e11dc7e.png)
![output_1-1](https://user-images.githubusercontent.com/2088817/54967874-b8d1ac80-4fb3-11e9-824a-3e628f78e044.png)
![output_1-2](https://user-images.githubusercontent.com/2088817/54967875-b8d1ac80-4fb3-11e9-9a2f-997ba4f54194.png)
It seems like a kind of mixture with leopard's body and tiger's face.

**Comment by thomwolf** [comment]
Oh, how did you get those?

**Comment by lukhy** [comment]
I had the same finding with you.
Did you just use "cat" as input name? The one_hot_from_names function uses NLTK to find the relevant labels in imagenet-1000 with your input names. You can check https://gist.github.com/yrevar/942d3a0ac09ec9e5eb3a and your one_hot vector to see what the original label is for input "cat".
Or if you just want to generate some normal cat, try using "Persian cat" as input name.

**Comment by eeyrw** [comment]
> Oh, how did you get those?

I just changed coffee into cat in the demo and ran it three times.


> I had the same finding with you.
> Did you just use "cat" as input name? The one_hot_from_names function uses NLTK to find the relevant labels in imagenet-1000 with your input names. You can check https://gist.github.com/yrevar/942d3a0ac09ec9e5eb3a and your one_hot vector to see what the original label is for input "cat".
> Or if you just want to generate some normal cat, try using "Persian cat" as input name.

I see. But I like this "cat" more.
